packages/ppx_rapper_lwt/ppx_rapper_lwt.3.0.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Lwt support for ppx_rapper"
+maintainer: ["Roddy MacSween <github@roddymacsween.co.uk>"]
+authors: ["Roddy MacSween <github@roddymacsween.co.uk>"]
+license: "MIT"
+homepage: "https://github.com/roddyyaga/ppx_rapper"
+doc: "https://github.com/roddyyaga/ppx_rapper"
+bug-reports: "https://github.com/roddyyaga/ppx_rapper/issues"
+depends: [
+  "ocaml" {>= "4.07"}
+  "dune" {>= "2.0.1"}
+  "ppx_rapper"
```

Comment:
If you're planning to release the three packages at the same time, could you sync up the version number as follow:
```
  "ppx_rapper" {= version}
```